Examples/doc-example-apis/readme.swift

```diff
@@ -0,0 +1,35 @@
+import Momento
+
+func main() async {
+  print("Running Momento Cache example!")
+  let cacheName = "example-cache"
+
+  var creds: CredentialProviderProtocol
+    do {
+        creds = try CredentialProvider.fromEnvironmentVariable(envVariableName: "MOMENTO_API_KEY")
+    } catch {
+        print("Error establishing credential provider: \(error)")
+        return
```

Comment:
Errors need to exit with a nonzero exit code so CI can flag a failure.
```suggestion
        exit(1)
```

Reply:
I don't think this file is getting run by ci, I'll move it to a new folder first and exit(1) there